fix skip issue

Marie59 opened the pull request:
@bgruening Like that ?
@pokapok-vracape ping

Marie59 committed:
fix skip issue

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ The feedstock has no `.ci_support` files and thus will not build any packages.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/15752359122](https://github.com/conda-forge/conda-forge-webservices/actions/runs/15752359122). Examine the logs at this URL for more detail.</sub>

bgruening commented:
@conda-forge-admin please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/15752253054](https://github.com/conda-forge/conda-forge-webservices/actions/runs/15752253054). Examine the logs at this URL for more detail.</sub>

bgruening committed:
Update meta.yaml

bgruening commented:
@conda-forge-admin please rerender

Commit:
MNT: Re-rendered with conda-build 25.5.0, conda-smithy 3.50.1, and conda-forge-pinning 2025.06.19.05.31.58

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

bgruening committed:
add skip back

bgruening commented:
@conda-forge-admin please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ `noarch` packages can't have skips with selectors. If the selectors are necessary, please remove `noarch: generic`, or selector on line 16:
  skip: true  # [win]



<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/15752642913](https://github.com/conda-forge/conda-forge-webservices/actions/runs/15752642913). Examine the logs at this URL for more detail.</sub>

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/15752648976](https://github.com/conda-forge/conda-forge-webservices/actions/runs/15752648976). Examine the logs at this URL for more detail.</sub>

bgruening committed:
Update meta.yaml

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

bgruening committed:
Update meta.yaml

Marie59 commented:
Hi @conda-forge/r should I open A new PR to create a new feedstock and try to follow this https://github.com/conda-forge/admin-requests?tab=readme-ov-file#add-a-package-output-to-a-feedstock ?

Sorry for the issue, thanks for your help !

ocefpaf committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.31.15.50.32

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.9

ocefpaf commented:
In theory https://github.com/conda-forge/admin-requests/pull/1735 should fix this.

ocefpaf commented:
No idea what is happening :-/

Maybe we should re-add it via staged-recipes. @Marie59 I'll re-open your PR, but can you make the meta.yaml look exactly like this one?

ocefpaf commented:
I refreshed the token in https://github.com/conda-forge/admin-requests/pull/1735, tried to rerender the original feedstock, and now trying to re-submit. Yet we are hitting:

```
validation results:
{
  "noarch/r-dmfiller-2.6-r44hd8ed1ab_0.conda": false
}
NOTE: Any outputs marked as False are not allowed for this feedstock. See https://conda-forge.org/docs/maintainer/infrastructure/#output-validation-and-feedstock-tokens for information on how to address this error.

##[error]Bash exited with code '1'.
```

Not sure what to do next :-/

@isuruf, can you see anything obvious that I am missing here?

ocefpaf commented:
We can use a new name, but that is not recommended. If nothing works, let's use `r-dmfiller-argo` as suggested in the previous commits of this PR.

mfansler commented:
@ocefpaf maybe you need to add a new feedstock output (`r-dmfiller`) via **admin-requests**?

ocefpaf commented:
> @ocefpaf maybe you need to add a new feedstock output (`r-dmfiller`) via **admin-requests**?

If we are renaming, yes. But I restored the original name here and refreshed the token. It should have been enough.

mfansler commented:
Ah sorry. I only saw the PR for token refresh not for adding output. Since it never pushed an artifact at creation time, I suspect it never registered permission to publish even under the default package name.

ocefpaf commented:
Yeah, conda-forge changed a lot and the feedstock registration is quite more complex than it was a few years ago. Hopefully someone from @conda-forge/core that worked on these changes can help us here. I would like to avoid creating a new name for this package if possible.

beckermr commented:
The output was missing from the feedstock outputs mapping, as the error in the build indicated. I added it via https://github.com/conda-forge/admin-requests/pull/1738 and now the build passes. Proceed as usual!

ocefpaf commented:
I wonder if it got removed due to the empty build, or maybe a buf somewhere. Thanks for sorting it out.

Marie59 commented:
Thanks a lot @ocefpaf @mfansler and @beckermr for your help !! 🥳
